spec/models/concerns/effective_settings_concern_spec.rb

```diff
@@ -11,7 +12,29 @@
     component_host.components.select(&:enabled_by_default?)
   end
   let(:default_enabled_component_ids) { default_enabled_components.map { |c| c.key.to_s } }
-  subject { Instance::Settings::Effective.new(settings, component_host) }
+
+  let(:parent_class) do
+    Class.new do
+      include SettingsConcern
+
+      def initialize(settings)
+        @settings = settings
+      end
+    end
+  end
```

Comment:
@lowjoel I've updated here.

Do you know any other ways to let me define this class more concisely ?